File: 240913/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.py
def is_number(n):
    sumN = 0;
    while n > 0:
        sumN += n % 10
        n = n // 10
    return sumN

n = int(input())
        
if is_number(n) % 5 == 0 and n % 2 == 0:
    print("Yes")
else:
    print("No")

File: 240913/짝수이면서 합이 5의 배수인 수/an-even-number-with-a-multiple-of-5-in-the-sum.c
#include <stdio.h>

int is_number(int n) {
    int sumN = 0;
    while (n > 0) {
        sumN += n % 10;
        n /= 10;
    }
    return sumN;
}

int main() {
    int n;
    scanf("%d", &n);
    if (n % 2 == 0 && is_number(n) % 5 == 0) {
        printf("Yes");
    }
    else {
        printf("No");
    }
    return 0;
}